types/Utils.ts

```diff
@@ -9,5 +9,5 @@ type Join<K, P> = K extends string | number
 export type Leaves<T> = [10] extends [never]
   ? never
   : T extends object
-  ? { [K in keyof T]-?: Join<K, Leaves<T[K]>> }[keyof T]
-  : '';
+    ? { [K in keyof T]-?: Join<K, Leaves<T[K]>> }[keyof T]
```

Comment:
Unrelated, but prettier was not happy with this file.

Reply:
Unfortunately I don't think we ever figured this out. @joe-yeager For context this particular line seems to format differently for different people and we haven't been able to figure out why

Reply:
@brandenrodgers @camden11 @kemmerle 
How do you all feel about adding a prettier ignore block around this?  It's my least favorite solution in these cases, but it feels better than committing with the `-n` flag

Reply:
Fine by me, definitely better than it going back and forth. Feel free to include that as part of this PR